plugins/templates/create.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const hoek = require('hoek');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/templates',
```

Comment:
{id} coincidentally is a number as well as what distinguishes uniqueness. There isn't an alternative way to distinguish one unique build from another, besides its build number. There cannot be two screwdriver-cd/screwdriver/builds/1. 

Templates are different. The characteristics that define a template from another are know before they do the create. What makes a template unique is the name and the version; there cannot be another instance of the same combination.

Look at this repository: screwdriver-cd/screwdriver. Github associates an {id} number to this repository too; where is it?